libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/MapboxTripNotification.kt

```diff
@@ -333,9 +334,7 @@ class MapboxTripNotification constructor(
     }
 
     companion object {
-        private var notificationActionButtonChannel = Channel<NotificationAction>(1)
-        fun getNotificationActionButtonChannel(): ReceiveChannel<NotificationAction> =
-            notificationActionButtonChannel
+        var notificationActionButtonChannel = Channel<NotificationAction>(1)
```

Comment:
do we need to make it public?

Reply:
Either way works because this is created via reflection but for convenience is easier if we make it public because we will avoid `isAccessible = true`